packages/flutter/lib/src/services/hardware_keyboard.dart

```diff
@@ -670,22 +672,26 @@ enum KeyDataTransitMode {
   ///
   /// If the current transit mode is [rawKeyData], the raw key data is converted
   /// to both [KeyMessage.events] and [KeyMessage.rawEvent].
+  ///
+  /// Raw data will be deprecated in the future, and this enum along with it.
```

Comment:
I don't think we need an announcement-to-deprecation. It's the same as documenting a design that is not implemented yet. Especially since `KeyDataTransitMode` is basically only used internally.